spec/controllers/concerns/hyrax/works_controller_behavior_spec.rb

```diff
@@ -84,10 +85,16 @@
         expect(response).to be_successful
       end
 
-      it 'renders a form' do
+      it 'assigns a change_set as the form' do
+        get :new
+
+        expect(assigns[:form]).to be_a Valkyrie::ChangeSet
+      end
+
+      it 'renders form' do
         get :new
 
-        expect(assigns[:form]).to be_kind_of Hyrax::WorkForm
+        expect(controller).to render_template('hyrax/base/new')
```

Comment:
As this is an open question on what the form builder will return, we are looking to test does the page render (instead of the assumption of the type of object that is assigned). Note the above `it 'assigns a change_set as the form'` replicates the behavior.